### PR TITLE
feat(breadcrumb/basic): restyling

### PR DESCRIPTION
**Pull request by JoanClaret**
feat(breadcrumb/basic): restyling

### CODE REVIEW

**Review by naxhh — CHANGES_REQUESTED** [review]
NIce work!

But you should add a BREAKING CHANGE in there.. because component props have changed

**Comment by JoanClaret** [comment]
yes, but at this moment, nobody is using this component. that's why i did not do it.

**Comment by JoanClaret** [comment]
added sass variables in order to be able to customize from theme
https://github.com/SUI-Components/theme-basic/pull/68
https://github.schibsted.io/scmspain/frontend-mt--uilib-theme/pull/43

**Review by naxhh — APPROVED** [review]

File: components/breadcrumb/basic/src/index.js [hunk 1]
```diff
@@ -1,43 +1,58 @@
...
-    </ul>
-  )
+export default class BreadcrumbBasic extends Component {
+  constructor (...args) {
+    super(...args)
+    this._breadcrumb = null
+  }
+
+  _expandBreadcrumb = () => {
+    this._breadcrumb.classList.add('is-expanded')
```
Comment (midudev) [inline components/breadcrumb/basic/src/index.js]: Why not using state for that?

File: components/breadcrumb/basic/src/index.js [hunk 2]
```diff
@@ -1,43 +1,58 @@
...
-          </Link>
-          { index < numLinks && <IconAngle svgClass='sui-BreadcrumbBasic-icon' /> }
-        </li>
-      )}
-    </ul>
-  )
+export default class BreadcrumbBasic extends Component {
+  constructor (...args) {
+    super(...args)
+    this._breadcrumb = null
```
Comment (midudev) [inline components/breadcrumb/basic/src/index.js]: You just could do:

```
_breadcrumb = null
```

And you remove the constructor.

File: components/breadcrumb/basic/src/index.js (hunk 1 above)
Comment (JoanClaret) [reply components/breadcrumb/basic/src/index.js]: i thought that using refs would be better option, but i might be wrong.
fixed!

File: components/breadcrumb/basic/src/index.js (hunk 2 above)
Comment (JoanClaret) [reply components/breadcrumb/basic/src/index.js]: actually, since we are now using state, we don't need to initialize this value.

**Review by midudev — APPROVED** [review]

**Pull request merged**